package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "bundle": "webpack --progress --colors",
     "dev": "webpack --progress --colors --watch",
-    "test": "istanbul cover --include-all-sources jasmine-node tests"
+    "test": "eslint src/ && istanbul cover --include-all-sources jasmine-node tests"
```

Comment:
Don't you want to lint both src/ and tests/ ? I usually see people just `eslint .` and then use ignore to ignore the other dirs